src/mocks/handlers.tsx

```diff
@@ -0,0 +1,37 @@
+// src/mocks/handlers.js
```

Comment:
In plain Typescript files without any JSX elements we don't need to support it, so the file extension should be `browser.tsx`.